srst2/0.2.0/Dockerfile

```diff
@@ -0,0 +1,61 @@
+FROM ubuntu:xenial as app
+
+# for easy upgrade later. ARG variables only persist at build time
+ARG SRST2_VER=0.2.0
+
+LABEL base.image="ubuntu:xenial"
+LABEL dockerfile.version="1"
+LABEL software="SRST2"
+LABEL software.version="v0.2.0"
+LABEL description="Short Read Sequence Typing for Bacterial Pathogens"
+LABEL website="https://github.com/katholt/srst2"
+LABEL license="https://github.com/katholt/srst2/blob/master/LICENSE.txt"
+LABEL maintainer="Holly Halstead"
+LABEL maintainer1.email="holly.halstead@doh.wa.gov"
+
+
+# install dependencies; cleanup apt garbage
+RUN apt-get update && apt-get install -y \
+python2.7 \
+python-pip \
+python-scipy \
+python-biopython \
+make \
+libc6-dev \
+g++ \
+zlib1g-dev \
+build-essential \
+git \
+libx11-dev \
+xutils-dev \
+zlib1g-dev \
+python-pip \
+bowtie2=2.2.6-2 \
+curl \
+libncurses5-dev \
+unzip \
+wget  \
+locate
```

Comment:
This looks great, but I noticed that there are some tools that you're installing twice (python-pip). Is there a reason for this?
Also, to save on image size could you change line 18 to 
```
RUN apt-get update && apt-get install -y --no-install-recommends \
```

And remove files that are no longer needed? 
(replace line 38 with)
```
locate && \
 apt-get autoclean && rm -rf /var/lib/apt/lists/*
```

Reply:
Changed! python-pip does not install cleanly with "apt-get install -y --no-install-recommends" but the other dependencies do. The section referenced above now installs python-pip after installing the other dependencies with "apt-get install -y --no-install-recommends" (and only once ;) ):

`RUN apt-get update && apt-get install -y --no-install-recommends \
python2.7 \
python-scipy \
python-biopython \
make \
libc6-dev \
g++ \
zlib1g-dev \
build-essential \
git \
libx11-dev \
xutils-dev \
zlib1g-dev \
bowtie2=${BOWTIE2_VER} \
curl \
libncurses5-dev \
unzip \
wget  \
locate && \
apt-get install -y \
python-pip && \
apt-get autoclean && rm -rf /var/lib/apt/lists/*`

Reply:
python-pip does not install smoothly with `apt-get install -y --no-install-recommends`, so I have used that commands with the other dependencies but kept `apt-get install -y` for the python-pip installation. I included your other suggestions as well to clean up the files which are no-longer needed and only have python-pip included once:

```
# install dependencies; cleanup apt garbage
RUN apt-get update && apt-get install -y --no-install-recommends \
python2.7 \
python-scipy \
python-biopython \
make \
libc6-dev \
g++ \
zlib1g-dev \
build-essential \
git \
libx11-dev \
xutils-dev \
zlib1g-dev \
bowtie2=${BOWTIE2_VER} \
curl \
libncurses5-dev \
unzip \
wget  \
locate && \
apt-get install -y \
python-pip && \
apt-get autoclean && rm -rf /var/lib/apt/lists/*
```